kie-remote/src/main/java/org/kie/remote/command/UpdateKJarCommand.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.remote.command;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+public class UpdateKJarCommand extends AbstractCommand implements VisitableCommand, RemoteCommand, Serializable {
+
+    private String kJarGAV;
+    private String groupID, artifactID, version;
+
+    public UpdateKJarCommand(String kjarGAV){
+        super(UUID.randomUUID().toString());
+        this.kJarGAV = kjarGAV;
+        String parts[]= kjarGAV.split(":");
```

Comment:
What about a utils method/class to do GAV management? This snippet is everywhere

Reply:
Yep, good idea

Reply:
Please use GAVUtils here too